src/forklift/models.py

```diff
@@ -138,6 +140,12 @@ def get_report(self):
                 'message': self.success[1],
                 'crates': [crate.get_report() for crate in self._crates]}
 
+    def send_email(self, to, subject, body):
```

Comment:
i don't think it's the best idea to have a method with the same name as an imported method. On second thought, why create a method with the same signature?

What do you think about 
```py
importing message

...

self.mailman = message

...

self.mailman.send_email(to, subject, body)
```

the `sendEmails` guard can go in the messages module? or as an optional parameter to the `send_email` method?

I'm not sure why we need a wrapper. Can you explain?